src/main/webapp/scripts/tutorial.js

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Function to set the tutorial in home.html
+ * @param {boolean} vendorIsLogged
+ */
+function handleTutorialContent(vendorIsLogged) {
+ (vendorIsLogged) ? setTutorialInDOM('vendor') : setTutorialInDOM('customer');
+}
+
+/** @param {string} tutorialFile */
+function setTutorialInDOM(tutorialFile) {
+  $('#tutorial-content').load(`tutorials/${tutorialFile}.html`);
+}
```

Comment:
Will this file be expanded upon in time? I'm just curious cause I feel like these two methods could be boiled down to one, and I'm all for separating duties etc but not sure this needs a script to itself?

Reply:
It may, depending on new features or functionalities of the tutorial!